app/models/user.py

```diff
@@ -21,3 +21,14 @@ def set_password(self, password):
 
     def check_password(self, password):
         return check_password_hash(self.password_hash, password)
+    
+    def get_item_names(self):
+        return [item.name for item in self.items]
+    
+    def to_dict(self):
```

Comment:
This function isn't necessary, the json provider in /app/utils/json_provider.py enhances the jsonify call to automatically deserialize items in a hash

Reply:
to_dict, that is. I realized that the preview in github makes it unclear which line I put the comment on.